packages/measure-tools/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "@bentley/measure-tools-react",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Frontend framework and tools for measurements",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/measure-tools"
+  },
+  "main": "lib/measure-tools.js",
+  "typings": "lib/measure-tools",
+  "scripts": {
+    "build": "npm run copy:assets && npm run pseudolocalize && tsc 1>&2 && npm run webpack:tests",
+    "copy:assets": "cpx \"./src/**/*.*css\" \"./lib\" && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "watch": "tsc --w",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "lint": "tslint --project . 1>&2",
+    "test": "node ./node_modules/@bentley/certa/bin/certa.js -c certa.json -r chrome",
+    "cover": "nyc npm test",
+    "madge": "madge --circular --extensions ts,tsx src",
+    "webpack:tests": "cross-env NODE_OPTIONS=--max-old-space-size=4096 webpack --config webpack.config.js 1>&2"
+  },
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
```

Comment:
Can you add `iTwin` and `iTwinjs` to the tags? I'll probably do that for all the others as well.